tox-python-versions

```diff
@@ -1,4 +1,5 @@
-3.6.12
 3.7.9
 3.8.6
 3.9.0
+3.10.9
+3.11.1
```

Comment:
I dunno how these are supposed to be chosen~ In my projects, I've never pinned the patch-level version of python.

Reply:
Yeah I'd be inclined to only list minor versions too, will check/update.